DMCompiler/DM/Builders/DMExpressionBuilder.cs

```diff
@@ -401,6 +401,11 @@ private static DMExpression BuildIdentifier(DMASTIdentifier identifier, DMObject
                         var global = new GlobalField(identifier.Location, globalVar.Type, globalId.Value, globalVar.ValType);
                         return global;
                     }
+
+                    var field = dmObject?.GetVariable(name);
+                    if (field != null) {
+                        return new Field(identifier.Location, field, field.ValType);
+                    }
```

Comment:
This allows you to also reference a non-static var.
```dm
/obj/var/static/b = a
/obj/var/a = 1
```
This needs to verify that it's constant.
```suggestion
                    var field = dmObject?.GetVariable(name);
                    if (field != null) {
                        if (field.IsConst)
                            return new Field(identifier.Location, field, field.ValType);

                        return BadExpression(WarningCode.BadExpression, identifier.Location,
                            "Var \"{name}\" cannot be used in this context");
                    }
```